dali/pipeline/operators/resize/resize_base.h

```diff
@@ -60,6 +62,19 @@ class DLL_PUBLIC ResizeBase : public ResamplingFilterAttr {
     kernels::KernelContext context;
     kernels::KernelRequirements requirements;
     kernels::ScratchpadAllocator scratch_alloc;
+    unique_ptr<void, void(*)(void*)> instance = { nullptr, free };
+
+    template <typename Kernel, typename... Args>
+    Kernel &KernelInstance(Args&&... args) {
+      auto deleter_lambda = [](void *ptr) {
```

Comment:
I wonder what's the role of this deleter?

Reply:
1. To delete an object of arbitrary type.
2. As a type tag.
This whole `unique_ptr<void` is a bit like `any`, but `any` cannot be used with non-copyable types. If it could, I'd use it - I actually tried, but got static_assert :(

Reply:
Oh sorry I missed than instance is field of the struct, forget my question :)